### ISSUE TITLE
Librem Key with X230

### ISSUE THREAD

**Issue by PowerPress** [osresearch/heads#459]
Will the Librem Key work the X230 laptop and flash green/red depending on tampering?
https://puri.sm/posts/the-librem-key-makes-tamper-detection-easy/

**Comment by tlaurion** [comment]
Yes. Read [here](https://github.com/osresearch/heads/pull/426#issuecomment-418136881) for current limitations.

**Comment by tlaurion** [comment]
@osresearch : can close.
